filebeat/magefile.go

```diff
@@ -82,8 +82,8 @@ func Package() {
 	mage.UseElasticBeatPackaging()
 	customizePackaging()
 
-	mg.Deps(Update, prepareModulePackaging)
-	mg.Deps(CrossBuild, CrossBuildXPack, CrossBuildGoDaemon)
+	//mg.Deps(Update, prepareModulePackaging)
```

Comment:
Commented out on purpose?

Reply:
Good catch, no, that was for faster debugging.